src/current/_includes/releases/v24.2/v24.2.4.md

```diff
@@ -0,0 +1,95 @@
+## v24.2.4
+
+Release Date: October 17, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+<h3 id="v24-2-4-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Updated the cluster setting [`changefeed.sink_io_workers`]({% link v24.2/cluster-settings.md %}#setting-changefeed-sink-io-workers) with all the [changefeed sinks]({% link v24.1/changefeed-sinks.md %}) that support the setting. [#130374][#130374]
+- Added two network metrics, `changefeed.network.bytes_in` and `changefeed.network.bytes_out`. [#130578][#130578]
+
+  These metrics track the number of bytes sent by individual [changefeeds]({% link v24.2/change-data-capture-overview.md %}) to the following sinks: 
+	- [Kafka sinks]({% link v24.2/changefeed-sinks.md %}#kafka). If [child metrics are enabled]({% link v24.2/cluster-settings.md %}#setting-server-child-metrics-enabled), the metric will have a `kafka` label. 
+	- [Webhook sinks]({% link v24.2/changefeed-sinks.md %}#webhook-sink). If child metrics are enabled, the metric will have a `webhook` label. 
+	- [Pub/Sub sinks]({% link v24.2/changefeed-sinks.md %}#google-cloud-pub-sub). If child metrics are enabled, the metric will have a `pubsub` label. 
+	- [SQL sink]({% link v24.2/changefeed-for.md %}). If child metrics are enabled, the metric will have a `sql` label.
+- Added a `changefeed.total_ranges` metric that can be used to monitor the number of ranges that are watched by [changefeed]({% link v24.2/change-data-capture-overview.md %}) aggregators. It shares the same polling interval as `changefeed.lagging_ranges`, which is controlled by the existing `lagging_ranges_polling_interval` option. [#130982][#130982]
+- Disambiguated [metrics]({% link v24.2/essential-metrics-self-hosted.md %}) and [logs]({% link v24.2/logging-overview.md %}) for the two buffers used by the KV feed. The following metrics now have a suffix indicating which buffer they correspond to: `changefeed.buffer_entries.*`, `changefeed.buffer_entries_mem.*`, or `changefeed.buffer_pushback_nanos.*`. The old versions are retained for backward compatibility, though using the new format is recommended. [#131419][#131419]
+- Added timers around key parts of the [changefeed]({% link v24.2/change-data-capture-overview.md %}) pipeline to help debug feeds experiencing issues. The `changefeed.stage.{stage}.latency` metrics now emit latency histograms for each stage. The metric respects the changefeed `scope` label for debugging specific feeds. [#131372][#131372]
+
+<h3 id="v24-2-4-sql-language-changes">SQL language changes</h3>
+
+- The [session variable]({% link v24.2/set-vars.md %}) `enforce_home_region_follower_reads_enabled` is now deprecated, and will be removed in a future release. The related session variable `enforce_home_region` remains supported and is not deprecated. [#129587][#129587]
```

Comment:
Is it deprecated in favor of something else or just period?

Reply:
@michae2 Any thoughts/suggestions on this?

Reply:
TYVM

```suggestion
- The [session variable]({% link v24.2/set-vars.md %}) `enforce_home_region_follower_reads_enabled` is now deprecated, in favor of `enforce_home_region`. The deprecated variable will be removed in a future release. [#129587][#129587]
```